zuul.d/jobs.yaml

```diff
@@ -478,3 +478,48 @@
     run: playbooks/packer-functional-devstack/run.yaml
     vars:
       packer_version: 1.2.5
+
+- job:
+    name: docker-machine-master-functional-opentelekomcloud
```

Comment:
Like packer, we should test specific version docker-machine against public cloud, for example, v0.15.0, and test master docker-machine with devstack.

Reply:
OK